src/components/shared/passphraseRenderer/passphraseRenderer.css

```diff
@@ -0,0 +1,123 @@
+@import '../../../app/variables.css';
+@import '../../../app/mixins.css';
+
+.header {
+  @mixin contentLargest bold;
+
+  color: var(--color-maastricht-blue);
+  display: flex;
+  margin: 0;
+}
+
+.subheader {
+  @mixin contentLargest;
+
+  text-align: left;
+  margin: 15px 0;
+  color: var(--color-slate-gray);
+}
+
+.passphraseContainer {
+  width: 100%;
+  background-color: var(--color-white);
+  border: 1px solid var(--dark-border-color);
+  flex-wrap: wrap;
+  padding: 12px 16px;
+  box-sizing: border-box;
+  counter-reset: indicator;
+
+  & .inputRow {
+    text-align: left;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  & .inputContainer {
+    margin: 6px 0;
+    display: flex;
+    justify-content: left;
+  }
+
+  & .inputValue {
+    @mixin contentLargest;
+
+    text-align: left;
+
+    &::before {
+      counter-increment: indicator;
+      content: counter(indicator) ". ";
+      color: var(--color-ghost);
+    }
+  }
+
+  & .default {
+    color: var(--color-maastricht-blue);
+  }
+
+  & .emptyWord,
+  & .emptyWordFocused {
+    cursor: pointer;
+    font-family: sans-serif;
+
+    &::before {
+      font-weight: var(--font-weight-bold);
+    }
+  }
+
+  & .emptyWord {
+    color: var(--color-ghost);
+
+    &::before {
+      color: var(--color-maastricht-blue);
+    }
+  }
+
+  & .emptyWordFocused {
+    color: var(--color-ultramarine-blue);
+
+    &::before {
+      color: var(--color-ultramarine-blue);
+    }
+  }
```

Comment:
Thank you, Sergio. This file looks much cleaner now.

Just one thing left. 💅 This one still repeats the same color and could be reduced to:
```css
  & .emptyWordFocused,
  & .emptyWordFocused::before {
    color: var(--color-ultramarine-blue);
  }
```